soap-attachments/3.0/_index.md

```diff
@@ -16,6 +16,13 @@ The goal of this release is to:
 The JDK version required will be aligned with Jakarta EE 10.
```

Comment:
Would be preferable if this stated what is supported -- Java 11 - 17 (I believe).